### PR TITLE
fa-IR locale with tests plus 1 fix for gl locale

### LINKED ISSUE #978: Persian language support
I'd love to start working on a locale for your great tool to support persian as well, for now I'll only translate words to Persian/Farsi counterparts and later on  I'll be more than glad to help you with my other submitted issue #977, to implement Jalali Support

Let me know dear

### PR DESCRIPTION
**Pull request by pkhodaveissi**
Please review and let me know what could be next

fixed #978

### CODE REVIEW

File: src/locale/fa-IR/_lib/localize/index.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  // If ordinal numbers depend on context, for example,
+  // if they are different for different grammatical genders,
+  // use `options.unit`:
+  //
+  //   var options = dirtyOptions || {}
+  //   var unit = String(options.unit)
+  //
+  // where `unit` can be 'year', 'quarter', 'month', 'week', 'date', 'dayOfYear',
+  // 'day', 'hour', 'minute', 'second'
```
Comment (kossnocorp) [inline src/locale/fa-IR/_lib/localize/index.js]: Please remove this comment as it supposed to be a helper to a locale contributor and doesn't make sense in this case.

File: src/locale/gl/_lib/formatRelative/index.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ var formatRelativeLocalePlural = {
   other: 'P'
 }
 
-export default function formatRelative (token, date, baseDate, options) {
-  if (date.getHours() !== 1) {
```
Comment (kossnocorp) [inline src/locale/gl/_lib/formatRelative/index.js]: Thanks 🙏

File: src/locale/fa-IR/_lib/localize/index.js (hunk 1 above)
Comment (pkhodaveissi) [reply src/locale/fa-IR/_lib/localize/index.js]: sure, will do in a day, i'm on road right now

Comment (kossnocorp) [reply src/locale/fa-IR/_lib/localize/index.js]: 👍

Comment (pkhodaveissi) [reply src/locale/fa-IR/_lib/localize/index.js]: just did, and please assign any issues relating my work here and I'll be happy to help

File: src/locale/gl/_lib/formatRelative/index.js (hunk 2 above)
Comment (pkhodaveissi) [reply src/locale/gl/_lib/formatRelative/index.js]: sure

**Comment by kossnocorp** [comment]
@pkhodaveissi sorry, I don't have enough time to do a proper review right now. It would help to move things forward if you also update `match/index.js` because I won't be able to publish the locale without it.

**Comment by kossnocorp** [comment]
Closed by #1113

**Pull request closed without merging**